### ISSUE TITLE
SRAO page in EBI OLS has dead links

### ISSUE THREAD

**Issue by only1chunts** [FAIRsharing/subject-ontology#72]
Hi, The EBI OLS page for the SRAO (https://www.ebi.ac.uk/ols/ontologies/srao) shows links to :

> Ontology IRI: [ http://www.fairsharing.org/ontology/subject/SRAO.owl](http://www.fairsharing.org/ontology/subject/SRAO.owl)
> Version IRI: [ http://www.fairsharing.org/ontology/subject/SRAO.owl/0.3.0](http://www.fairsharing.org/ontology/subject/SRAO.owl/0.3.0)
> 

both of which point to Error 404 pages.

**Comment by allysonlister** [comment]
You are correct - my apologies! There should be redirects in place - I will make a ticket within FAIRsharing. Thanks!

**Comment by allysonlister** [comment]
This has been resolved. Please let me know if you have any further questions!

Thanks :)
